src/app/admin/api-key/api-key-list/api-key-table/api-key-table.component.html

```diff
@@ -31,32 +31,53 @@
 
     <!-- Key Column -->
     <ng-container matColumnDef="key">
-      <th mat-header-cell *matHeaderCellDef>
+      <th *matHeaderCellDef mat-header-cell>
         {{ "API-KEY.KEY" | translate }}
       </th>
-      <td mat-cell *matCellDef="let element">
+      <td *matCellDef="let element" mat-cell>
         {{ element.key }}
       </td>
     </ng-container>
 
+    <!--  Expiration Column  -->
+    <ng-container matColumnDef="expiresOn">
+      <th *matHeaderCellDef mat-header-cell mat-sort-header>
+        {{ "API-KEY.EXPIRES-ON" | translate }}
+      </th>
+      <td *matCellDef="let element" mat-cell>
+        {{ (element.expiresOn | dateOnly) ?? '-' }}
```

Comment:
Can it be done so it doesnt always translate to english? If it's not easy then it might not matter